src/main/java/frc/robot/RobotContainer.java

```diff
@@ -25,31 +29,47 @@
  */
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
-  private final SwerveSubsystem swerveSubsystem = new SwerveSubsystem();
-  private final EverybotIntakeSubsystem intakeSubsystem = new EverybotIntakeSubsystem();
-
-  private SimulationSubsystem simulationSubsystem;
-  
-  private final CameraSubsystem cameraSubsystem = new CameraSubsystem();
-
+  private final SwerveSubsystem swerveSubsystem;
+  private final SimulationSubsystem simulationSubsystem;
+  private final EverybotIntakeSubsystem intakeSubsystem;
+  private final CameraSubsystem cameraSubsystem;
   private static BatterySubsystem batterySubsystem;
 
-  private Auton auton = new Auton(swerveSubsystem);
+  private Auton auton;
+
+  private DataLog robotSubsystemsLog = DataLogManager.getLog();
+  private StringLogEntry swerveEnabledLog = new StringLogEntry(robotSubsystemsLog, "Swerve Enabled");
+  private StringLogEntry simulationEnabledLog = new StringLogEntry(robotSubsystemsLog, "Simulation Enabled");
+  private StringLogEntry intakeEnabledLog = new StringLogEntry(robotSubsystemsLog, "Inatake Enabled");
+  private StringLogEntry cameraEnabledLog = new StringLogEntry(robotSubsystemsLog, "Camera Enabled");
+  private StringLogEntry batteryEnabledLog = new StringLogEntry(robotSubsystemsLog, "Battery Enabled");
```

Comment:
To make this easier, we can have one StringLogEntry and log status to that.

So like:  
private StringLogEntry subsystemEnabledLog = new StringLogEntry(robotSubsystemsLog, "/robot/enabled");

And then below you can do:
subsystemEnabledLog.append(swerveSubsystem == null ? "Swerve Disabled" : "Swerve Enabled");

Reason why one log would be preferred is so that you can easily search for enabled, and see all log entries